cli/npm/managed/mod.rs

```diff
@@ -387,11 +388,6 @@ impl ManagedCliNpmResolver {
     self.resolution.add_package_reqs(packages).await?;
     self.fs_resolver.cache_packages().await?;
 
-    // If there's a lock file, update it with all discovered npm packages
-    if let Some(lockfile) = &self.maybe_lockfile {
-      self.lock(&mut lockfile.lock());
-    }
```

Comment:
Where is lockfile handling done now?

Reply:
This was done twice by accident. It's now updated only within the update queue of cli/npm/managed/resolution.rs (`populate_lockfile_from_snapshot`).